CDP4Composition/LayoutGroupNames.cs

```diff
@@ -0,0 +1,29 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
```

Comment:
Header different. Probably copied from a very old file. We only fix the old files when we touch them, but eventually all source will change to AGPL license header.